click/index.html

```diff
@@ -12,6 +12,13 @@
     <script src="base.js"></script>
 	</head>
 	<body>
+    <ul>
+      <li><a href='file:///Users/Noel/dev/23-homework/events_lab/submit/index.html'>SUBMIT</a></li>
+      <li><a href='file:///Users/Noel/dev/23-homework/events_lab/ready/index.html'>READY</a></li>
+      <li><a href='file:///Users/Noel/dev/23-homework/events_lab/keypress/index.html'>KEYPRESS</a></li>
+      <li><a href='file:///Users/Noel/dev/23-homework/events_lab/click/index.html'>CLICK</a></li>
```

Comment:
@noel-arce these kind of links shouldn't be used since they'll only work on your machine and only if you don't move the directory